<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrnGenerateDialog</class>
 <widget class="QDialog">
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox">
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Fix colors:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="LineEditWidget" name="firstFixColorLineEdit">
            <property name="charWidth">
             <number>4</number>
            </property>
            <property name="toolTip">
             <string>Number of extra maps to generate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LineEditWidget" name="lastFixColorLineEdit">
            <property name="charWidth">
             <number>4</number>
            </property>
            <property name="toolTip">
             <string>Number of extra maps to generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="shadeFixColorCheckBox">
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>These colors must not change</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="addRangeButtonLayout"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Palettes:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="LineEditWidget" name="paletteFileEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="paletteFileBrowseButton">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="addRangeButtonLayout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
